src/screens/WineManager/components/WineItem/index.tsx

```diff
@@ -110,17 +106,19 @@ const WineItem = ({
 
   return (
     <>
-      <div className="toolsListItem">
-        <span className="toolsTitleList">
-          {type} - {version}
-        </span>
-        <div className="toolsListDate">{date}</div>
-        <div className="toolsListSize">{renderStatus()}</div>
+      <div className="wineManagerListItem">
+        <span className="wineManagerTitleList">{version}</span>
+        <div className="wineManagerListDate">{date}</div>
+        <div className="wineManagerListSize">{renderStatus()}</div>
         <span className="icons">
           {!isInstalled && !isDownloading && !unZipping && (
             <DownIcon className="downIcon" onClick={() => install()} />
           )}
-          {(isDownloading || unZipping) && <StopIcon onClick={() => null} />}
+          {(isDownloading || unZipping) && (
+            <StopIcon
+              onClick={() => ipcRenderer.send('abortWineInstallation')}
```

Comment:
it doesn't need to pass the version to abort? what happens if more than one version is being downloaded and we click on abort?

Reply:
Good point. Will fix this later